portfolio/src/main/webapp/index.html

```diff
@@ -12,7 +12,7 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body onload="getComments();createMap();">
```

Comment:
is createmap defined? or is that from another branch?

Reply:
another branch that is merged into master!